packages/math-input/src/components/tabbar/__tests__/tabbar.test.tsx

```diff
@@ -1,65 +1,54 @@
-import {mount} from "enzyme";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import * as React from "react";
 
 import Tabbar from "../tabbar";
 
 describe("<Tabbar />", () => {
-    it("defaults to selecting the first item", () => {
+    it("renders the numbers page", () => {
         // Arrange
-        const wrapper = mount(
+        render(
             <Tabbar
-                items={["Numbers", "Geometry", "Operators"]}
-                onSelect={() => {}}
+                items={["Numbers"]}
+                selectedItem={"Numbers"}
+                onSelectItem={() => {}}
             />,
         );
 
         // Assert
-        expect(wrapper).toHaveState("selectedItem", 0);
-        const firstItem = wrapper.find("TabbarItem").first();
-        expect(firstItem).toHaveProp("itemState", "active");
+        expect(screen.getByRole("button", {name: "Numbers"}));
     });
 
-    it("selects the second item", () => {
+    it("renders all tabs", () => {
         // Arrange
-        const wrapper = mount(
+        render(
             <Tabbar
-                items={["Numbers", "Geometry", "Operators"]}
-                onSelect={() => {}}
+                items={["Numbers", "Extras", "Geometry", "Operators"]}
+                selectedItem={"Numbers"}
+                onSelectItem={() => {}}
             />,
         );
 
-        // Act
-        let secondItem = wrapper.find("TabbarItem").at(1);
-        secondItem.simulate("click");
-
         // Assert
-        expect(wrapper).toHaveState("selectedItem", 1);
-        const firstItem = wrapper.find("TabbarItem").at(0);
-        expect(firstItem).toHaveProp("itemState", "inactive");
-        // NOTE: we have to re-get the second item to get it's updated state
-        secondItem = wrapper.find("TabbarItem").at(1);
-        expect(secondItem).toHaveProp("itemState", "active");
+        expect(screen.getByRole("button", {name: "Numbers"}));
+        expect(screen.getByRole("button", {name: "Extras"}));
+        expect(screen.getByRole("button", {name: "Geometry"}));
+        expect(screen.getByRole("button", {name: "Operators"}));
```

Comment:
Does calling `expect()` without chaining an assertion do anything?  I think by the fact that you are using `getByXyz()` functions is enough of an assertion to fail the test if that element can't be found.

Reply:
Good call. These tests do work as-is, but it's more explicit to chain an expectation. Fixed.